skynet/modules/ttt/summaries/app.py

```diff
@@ -1,6 +1,9 @@
+import requests
```

Comment:
We don't use this anywhere else do we? In fact we use aiohttp and a small client wrapper around it. Let's pick one. IIRC requests is blocking, so not a good fit here.